litr-muxers/src/main/cpp/Logging.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2022 LinkedIn Corporation
+ * All Rights Reserved.
+ *
+ * Licensed under the BSD 2-Clause License (the "License").  See License in the project root for
+ * license information.
+ *
+ * Author: Ian Bird
+ */
+
+#ifndef LITR_LOGGING_H
+#define LITR_LOGGING_H
+
+#include <android/log.h>
+
+#define LOG_TAG "LiTrMuxers_JNI"
+#define LOGD(...) ((void)__android_log_print(ANDROID_LOG_DEBUG, LOG_TAG, __VA_ARGS__))
```

Comment:
Logging functions that can be used from C++ that will end up in Android's logcat.